xbmc/epg/EpgContainer.cpp

```diff
@@ -148,7 +149,7 @@ class CEPGContainerStartJob : public CJob
   }
 };
 
-void CEpgContainer::Start(bool bAsync)
+void CEpgContainer::Start(bool bAsync, bool bIsRestart /* = false */)
```

Comment:
that it's restarting is not actually of interest here in this method, because you're actually just suppressing the progress dialog. better name it like that, as that reflects what this parameter is doing